### DISCUSSION TITLE
man pages

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#171] [Porting]
See this - the top is attempting to view man pages for bash and less from the port and the bottom is the bash man page provided with z/OS OMVS

![image](https://user-images.githubusercontent.com/42328411/218117045-7526636c-8a85-4b0c-b9ae-553dd2403904.png)

**Comment by MikeFultonDev** [comment]
Thanks Lionel. We need to get man-db ported (it doesn't run yet) because the z/OS 'man' doesn't support modern man pages. Here's a list of issues: https://github.com/ZOSOpenTools/man-dbport/issues

**Comment by lbdyck** [comment]
just read that - ouch - but I'm confident you can do it 😃

**Comment by MikeFultonDev** [comment]
This should all be working now. The only caveat is when you have a tool that is both in z/OS Open Tools _and_ the base z/OS (such as cp). For tools only in one or the other, the latest 'man' should work.
